core/diagnostics/DiagnosticScenarios/Controllers/DiagnosticScenarios.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+
+namespace testwebapi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DiagScenarioController : ControllerBase
+    {
+        object o1 = new object();
```

Comment:
Line 15 and 16, should use `var`.

Reply:
I had to revert this.  `var` is not allowed at class scope.